### PR TITLE
idfv release from our latest tagged version, with test

### PR DESCRIPTION
**Pull request by gianpiero-bozza**

### CODE REVIEW

File: PrebidMobile/Constants.swift [hunk 1]
```diff
@@ -20,6 +20,11 @@ extension String {
     static let EMPTY_String = ""
 
     static let kIFASentinelValue = "00000000-0000-0000-0000-000000000000"
+    
+    // this vendor id is generated based on the bundle id, i.e.: "com.example.app1"
+    // and stays the same while the app is installed on the device.
+    // Source: https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor
+    static let kIFVSentinelValue = "4EE19465-23E0-455F-BA43-A4E962766CF0"
```
Comment (ondra-kormanik) [inline PrebidMobile/Constants.swift]: I dont understand a purpose of this. What does this id mean? If it is vendor id generated based on the budle id, then we cannot know it here in Prebid library.

File: tools/PrebidValidator/Dr.Prebid.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -671,7 +671,7 @@
 				"${BUILT_PRODUCTS_DIR}/PromisesObjC/FBLPromises.framework",
 				"${BUILT_PRODUCTS_DIR}/mopub-ios-sdk/MoPubSDK.framework",
 				"${BUILT_PRODUCTS_DIR}/nanopb/nanopb.framework",
-				"${PODS_XCFRAMEWORKS_BUILD_DIR}/OMSDK_Mopub/OMSDK_Mopub.framework/OMSDK_Mopub",
```
Comment (ondra-kormanik) [inline tools/PrebidValidator/Dr.Prebid.xcodeproj/project.pbxproj:674]: should it be here?

File: PrebidMobile/RequestBuilder.swift [hunk 3]
```diff
@@ -364,10 +364,13 @@ class RequestBuilder: NSObject {
         let lmtAd: Bool = !ASIdentifierManager.shared().isAdvertisingTrackingEnabled
         // Limit ad tracking
         deviceDict["lmt"] = NSNumber(value: lmtAd).intValue
+        var ifa:String = ""
```
Comment (ondra-kormanik) [inline PrebidMobile/RequestBuilder.swift]: `String` is redundant

**Review by ondra-kormanik — COMMENTED** [review]
I dont see that it does `We send the ATT consent status to PBS whenever it is available`. Does Prebid do it?

**Comment by gianpiero-bozza** [comment]
> I dont see that it does `We send the ATT consent status to PBS whenever it is available`. Does Prebid do it?

line 599 in RequestBuiler.swift implements that logic, already there if I'm not wrong. it wasn't added by us
if #available(iOS 14, *) {
    deviceExt["atts"] = ATTrackingManager.trackingAuthorizationStatus.rawValue
}

File: tools/PrebidValidator/Dr.Prebid.xcodeproj/project.pbxproj (hunk 2 above)
Comment (gianpiero-bozza) [reply tools/PrebidValidator/Dr.Prebid.xcodeproj/project.pbxproj:674]: I still see this in my local files and I have everything committed and pushed.. not sure why that line was removed.
I didn't remove it manually.. this file is Xcode proprietary, or something like that.

Comment (gianpiero-bozza) [reply tools/PrebidValidator/Dr.Prebid.xcodeproj/project.pbxproj:674]: after I reverted it and used "pod install" again this was modified again by pod install.

File: PrebidMobile/Constants.swift (hunk 1 above)
Comment (gianpiero-bozza) [reply PrebidMobile/Constants.swift]: completely removed. Reworked the test to be more generic and don't use specific values.
I found out that the string also changes depending on what device the tests are running on.
Thanks for pointing this out.

**Pull request merged**